tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -1,12 +1,13 @@
-import React, { useState } from 'react';
-import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet } from 'react-native';
 import { CalendarList } from 'react-native-calendars';
-import { BackButton } from '../components/BackButtonComponent';
+import { DayComponent } from '../components/DayComponent'
 import Selector from '../components/Selector';
 import Icon from 'react-native-vector-icons/MaterialIcons';
 import {Button} from 'react-native-elements';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import { STACK_SCREENS } from '../CalendarNavigator';
+import { GETYearData } from '../../services/CalendarService';
+import { VIEWS } from '../../services/utils/constants';
```

Comment:
This was probably from a merge, but importing VIEWS here conflicts with line 12 below. Suggest removing line 12-19 since it makes more sense to have VIEWS come from constants